frontend/src/v4/modules/viewpoints/viewpoints.sagas.ts

```diff
@@ -68,10 +68,7 @@ export function* generateViewpoint(teamspace, modelId, name, withScreenshot = fa
 	try {
 		const hiddenGeometryVisible = yield select(selectHiddenGeometryVisible);
 
-		const viewpoint = yield Viewer.getCurrentViewpoint({
-			teamspace,
-			model: modelId
-		});
+		const viewpoint = yield Viewer.getCurrentViewpoint();
```

Comment:
Since this was the only point in the function body using `teamspace` and `modelId`, I think you can remove their definition from the parameters

Reply:
I prefer to leave v4 code untouched as much as possible.